### PR TITLE
move webflow scripts to v44 (currently testing against testnet api)

### PR DESCRIPTION
**Pull request by samcode206**
[asana](https://app.asana.com/0/1189045082777889/1201198229752746/f)

todo: test on deploy preview for webflow

### CODE REVIEW

**Review by adamrturman — COMMENTED** [review]
@samcode206 - could you please include a link to the webflow deploy preview so we can verify that stats for the new denoms are displaying as expected?

File: kava-stats.js [hunk 1]
```diff
@@ -766,7 +788,33 @@ const setDisplayValueById = (cssId, value) => {
 };
 
 const updateDisplayValues = async (denoms) => {
-    const [
+  const data = {
```
Comment (samcode206) [inline kava-stats.js:791]: this data object makes it easier to destructive responses, with Promise.All it had to be in exact order which made it hard to keep track & add new endpoints.

File: hard-protocol-home.js [hunk 2]
```diff
@@ -144,6 +144,53 @@ const commonDenomMapper = (denom) => {
...
+    return cache.get(denom);  
+  };
+  const request = await fetch(BASE_URL + '/ibc/apps/transfer/v1/denom_traces/' + denom.replace("ibc/",""));
+  const ibcDenom = await request.json();
+  cache.set(denom, ibcDenom.denom_trace.base_denom);
+  return ibcDenom.denom_trace.base_denom; 
+}; 
+
+const normalizeDenoms = async (denomsList) => {
+  const readable = [];
```
Comment (adamrturman) [inline hard-protocol-home.js:160]:
```suggestion
  const readableDenoms = [];
```
and when used elsewhere similar to your naming convention for `readableParams`

File: hard-protocol-home.js [hunk 3]
```diff
@@ -144,6 +144,53 @@ const commonDenomMapper = (denom) => {
...
+  return readable;
+};
+
+const normalizeCollateralTypes = async (params, isPool = false) => {
+  const readableParams = []; 
+  for (const param of params){
+    if (param.collateral_type.includes("ibc")){
+      let readableCollateralType;
+      if (isPool){
+        readableCollateralType = await ibcDenomMapper(param.collateral_type.split(":")[0]) + ":usdx";
```
Comment (adamrturman) [inline hard-protocol-home.js:178]: Might be worth breaking `.split(':')[0] + ':usdx' but just a preference.

File: kava-protocol-home.js [hunk 4]
```diff
@@ -80,7 +80,7 @@ const displayInThousands = (value) => {
 }
 
 const isKavaNativeAsset = (denom) => {
-  return ['ukava-a', 'usdx', 'hard', 'ukava', 'hard-a', 'swp-a'].includes(denom)
+  return ['ukava-a', 'usdx', 'hard', 'ukava','kava-a', 'hard-a', 'swp-a'].includes(denom)
```
Comment (adamrturman) [inline kava-protocol-home.js:83]: When would we encounter a cdp type of `kava-a`?

File: kava-protocol-home.js [hunk 5]
```diff
@@ -231,7 +288,8 @@ const mapTotalSupplied = async (denoms, siteData) => {
     if (isKavaNativeAsset(denom)) {
       denomTotalSupply = suppliedAmountData[denom].amount;
     } else {
-      denomTotalSupply = bep3SupplyData[denom];
+      // denomTotalSupply = bep3SupplyData[denom];
```
Comment (adamrturman) [inline kava-protocol-home.js:291]: Is this intentionally left in?

**Review by adamrturman — DISMISSED** [review]

File: kava-protocol-home.js (hunk 4 above)
Comment (samcode206) [reply kava-protocol-home.js:83]: the new endpoints is returning kava for the denom name I believe I can check the exact one for you.

**Review by stephensprinkle — APPROVED** [review]

**Pull request merged**